src/protocol/messages/mod.rs

```diff
@@ -86,6 +86,16 @@ impl<W: Write, T: WriteVersionedType<W>> WriteVersionedType<W> for &T {
     }
 }
 
+/// A response body with metadata about the request.
```

Comment:
```suggestion
/// A response body with metadata about the request & response.
```

I think we can make this a bit more generic for future additions.

Reply:
Updated in [a12992f](https://github.com/influxdata/rskafka/pull/276/commits/a12992f6d3c7da192b0399e3462b89f7188b2758)